pages/privacy-basics.jsx

```diff
@@ -0,0 +1,194 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+var ActivitySection = require('../components/activity-section.jsx');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var Intro = React.createClass({
+  render: function () {
+    return (
+      <div>
+        <h1>Privacy Basics: Passwords, Tracking, and Data Retention</h1>
+        <section className="intro">
+          <Illustration
+          height={204} width={204}
+          src1x=" https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png"
+          src2x=" https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png"
+          alt=""
+          className="illustration-img-circle">
+            <h2>Learn how to safeguard your privacy online and develop an awareness of how companies and governments track and collect your data online.</h2>
+          </Illustration>
+        </section>
+      </div>
+    );
+  }
+});
+
+
+var LearningObjectives = React.createClass({
+  render: function () {
+    return (
+      <section className="row privacy-basics">
+        <div className="col-sm-12">
+          <h2>Learning Objectives</h2>
+          <p>
+            Learners will understand Web mechanics, security, and privacy as they analyze and reflect on common surveillance practices, as well as their own privacy habits.
+          </p>
+        </div>
+      </section>
+    );
+  }
+});
+
+
+var PrivacyBasics = React.createClass({
+  statics: {
+    pageTitle: 'Privacy Basics: Passwords, Tracking, and Data Retention',
+    pageClassName: 'privacy-basics'
+  },
+  sections: [
+    {
+      title: "Passwords",
+      activities: [
+        {
+          title: "Passwords Agenda",
+          image1x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
+          image2x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
```

Comment:
same, no need to have `image2x` if we don't have a retina version of the image :p this applies to other sections below

also it'll be rad if you can remove the extra space at the beginning :hamster: